.azuredevops/components/hipTensor.yml

```diff
@@ -10,21 +10,25 @@ parameters:
   default:
     - cmake
     - ninja-build
-    - composablekernel-dev
+    - python3-pip
     - git
 - name: rocmDependencies
   type: object
   default:
+    - rocm-cmake
     - llvm-project
+    - ROCR-Runtime
+    - clr
+    - rocminfo
+    - composable_kernel
 
 jobs:
 - job: hipTensor
   variables:
   - group: common
   - template: /.azuredevops/variables-global.yml
-  pool: ${{ variables.MEDIUM_BUILD_POOL }}
-  container:
-    image: ${{ variables.DOCKER_IMAGE_NAME }}:${{ variables.LATEST_DOCKER_VERSION }}
+  pool:
+    vmImage: ${{ variables.BASE_BUILD_POOL }}
```

Comment:
Let's change this back to medium build pool for deployment since base pool take 50+ minutes to build.

Reply:
OK, I changed it to BASE to avoid the traffic, will switch it back